01-arch/07-reputation.md

```diff
@@ -0,0 +1,41 @@
+## Reputation system
+
+\Gls{NeoFS} reputation system is a subsystem for calculating trust in \Gls{Node}. It is based on a reputation model for assessing trust, which is, in turn, based on the EigenTrust algorithm designed for peer-to-peer reputation management. The algorithm ensures that there is a uniquely defined manager (parent node) for each network participant at each specified time (epoch). Based on the information received from its child node and other managers, it iteratively puts a complex general (Global) Trust of the entire network into the applicable child node.
```

Comment:
I suggest adding few words on why do we need the reputation system in NeoFS and how it's supposed to be used.

Reply:
added some information about practical usage